shapely/constructive.py

```diff
@@ -128,33 +130,34 @@ def buffer(
 
     Examples
     --------
-    >>> buffer(Geometry("POINT (10 10)"), 2, quadsegs=1)
-    <pygeos.Geometry POLYGON ((12 10, 10 8, 8 10, 10 12, 12 10))>
-    >>> buffer(Geometry("POINT (10 10)"), 2, quadsegs=2)
-    <pygeos.Geometry POLYGON ((12 10, 11.414 8.586, 10 8, 8.586 8.586, 8 10, 8.5...>
-    >>> buffer(Geometry("POINT (10 10)"), -2, quadsegs=1)
-    <pygeos.Geometry POLYGON EMPTY>
-    >>> line = Geometry("LINESTRING (10 10, 20 10)")
+    >>> from shapely import LineString, Point, Polygon
+    >>> buffer(Point(10, 10), 2, quadsegs=1)
+    <shapely.Polygon POLYGON ((12 10, 10 8, 8 10, 10 12, 12 10))>
+    >>> buffer(Point(10, 10), 2, quadsegs=2)
+    <shapely.Polygon POLYGON ((12 10, 11.414 8.586, 10 8, 8.586 8.586, 8 10, 8.5...>
+    >>> buffer(Point(10, 10), -2, quadsegs=1)
+    <shapely.Polygon POLYGON EMPTY>
+    >>> line = LineString(((10, 10), (20, 10)))
     >>> buffer(line, 2, cap_style="square")
-    <pygeos.Geometry POLYGON ((20 12, 22 12, 22 8, 10 8, 8 8, 8 12, 20 12))>
+    <shapely.Polygon POLYGON ((20 12, 22 12, 22 8, 10 8, 8 8, 8 12, 20 12))>
     >>> buffer(line, 2, cap_style="flat")
-    <pygeos.Geometry POLYGON ((20 12, 20 8, 10 8, 10 12, 20 12))>
+    <shapely.Polygon POLYGON ((20 12, 20 8, 10 8, 10 12, 20 12))>
     >>> buffer(line, 2, single_sided=True, cap_style="flat")
-    <pygeos.Geometry POLYGON ((20 10, 10 10, 10 12, 20 12, 20 10))>
-    >>> line2 = Geometry("LINESTRING (10 10, 20 10, 20 20)")
+    <shapely.Polygon POLYGON ((20 10, 10 10, 10 12, 20 12, 20 10))>
+    >>> line2 = LineString(((10, 10), (20, 10), (20, 20)))
     >>> buffer(line2, 2, cap_style="flat", join_style="bevel")
-    <pygeos.Geometry POLYGON ((18 12, 18 20, 22 20, 22 10, 20 8, 10 8, 10 12, 18...>
+    <shapely.Polygon POLYGON ((18 12, 18 20, 22 20, 22 10, 20 8, 10 8, 10 12, 18...>
     >>> buffer(line2, 2, cap_style="flat", join_style="mitre")
-    <pygeos.Geometry POLYGON ((18 12, 18 20, 22 20, 22 8, 10 8, 10 12, 18 12))>
+    <shapely.Polygon POLYGON ((18 12, 18 20, 22 20, 22 8, 10 8, 10 12, 18 12))>
     >>> buffer(line2, 2, cap_style="flat", join_style="mitre", mitre_limit=1)
-    <pygeos.Geometry POLYGON ((18 12, 18 20, 22 20, 21.828 9, 21 8.172, 10 8, 10...>
-    >>> square = Geometry("POLYGON((0 0, 10 0, 10 10, 0 10, 0 0))")
+    <shapely.Polygon POLYGON ((18 12, 18 20, 22 20, 22 9.172, 20.828 8, 10 8, 10...>
```

Comment:
Not sure why, but I got slightly different values (macOS 10.15 / > GEOS 3.10)

Reply:
It's annoying when writing those examples on a mac, but you will need to keep the output as it was. We currently test the docstrings on one CI build (ubuntu python 3.10 build), and so it has to pass on that build. And currently this example is failing: https://github.com/shapely/shapely/runs/6824676810?check_suite_focus=true#step:17:54